scripts/changelog-check.js

```diff
@@ -1,34 +1,34 @@
 const { exec } = require("child_process");
 
-const changeLogCheck = () => {
-  exec(
-    "git diff origin/master --cached --name-only",
-    (error, stdout, stderr) => {
-      if (error || stderr) {
-        process.exit(1);
-      }
+const normalizePath = (path) => path.replace(/\\+/g, "/").trim().toLowerCase();
 
-      if (!stdout || stdout.includes("packages/excalidraw/CHANGELOG.MD")) {
-        process.exit(0);
-      }
+const IGNORED_PATHS = [
+  "src/excalidraw-app",
+  "packages/utils",
+  "CHANGELOG.md",
+  "README.md",
+].map(normalizePath);
 
-      const onlyNonSrcFilesUpdated = stdout.indexOf("src") < 0;
```

Comment:
this will help in early return so we should keep this

Reply:
performance-wise it's totally negligible and I think it's best to colocate the logic as much as possible

Reply:
true but just feels unnecessary to run the checks for non src files hence suggested keeping it. 
Rest all lgtm. Feel free to merge 👍